doc/odoc_for_authors.mld

```diff
@@ -297,7 +297,7 @@ v}
 {4:language_headers Language Headers}
 
 As of [odoc.2.2] it is possible to write blocks with explicit language headers.
-These blocks can be written using the enclosing tags [{@<language>[ ... ]}].
+These blocks can be written using the enclosing tags [{@<language>[ <code block> ]}].
```

Comment:
In general I like the consistent use of placeholder syntax, but specifically when you replace `...` I'm not fond of the result. The problem here is that the name of the placeholder is tautological, and at the same time slightly wrong at places -- here there is a confusion between "a code block" and "the content of a code block", I guess it should be `<code>` instead. The nice thing with named placeholders is that they carry meaning and can be referred to in the text; but in all examples that were using `...` before we don't really need either of these two aspects.

If it was my PR, I would keep the `...` unchanged.

Reply:
I was hesitant for this part: I went for changing the `...` first, but I just changed it back since you have a clearer opinion than me!